File: aspnet/GrabzIt/ScreenShotStatus.cs
using System;
using System.IO;
using System.Xml.Serialization;

namespace GrabzIt
{
    [Serializable]
    [XmlType(TypeName = "WebResult")]
    public class ScreenShotStatus
    {
        public bool Processing;
        public bool Cached;
        public bool Expired;
        public string Message;

        public ScreenShotStatus()
        {            
        }
    }
}

File: branches/version1/ruby/GrabzIt/lib/screenshotstatus.rb
class ScreenShotStatus
	def initialize
		@Processing = false
		@Cached = false
		@Expired = false
		@Message = ''
	end
	def processing
		@Processing
	end
	def processing=(value)
		@Processing = value
	end  
	def cached
		@Cached
	end
	def cached=(value)
		@Cached = value
	end  
	def expired
		@Expired
	end
	def expired=(value)
		@Expired = value
	end  
	def message
		@Message
	end
	def message=(value)
		@Message = value
	end  	
end
